core/lib/__tests__/richText.test.ts

```diff
@@ -0,0 +1,199 @@
+import { getPubValues } from "context-editor";
```

Comment:
These tests are for `parseRichTextForPubFieldsAndRelatedPubs` which is meant to take in form values (from either the form fill page or a pub editor) and transform them into `values` and `relatedPubs`. Ideally this is something that our `createPub` function can take, though the `relatedPub` portion is still being built out in other tickets.

Part of `parseRichTextForPubFieldsAndRelatedPubs` calls a function which is exported by the ContextEditor, and so that function is stubbed out. I also think that function is quite likely to change in the future.